frontend/src/workbench/blocks/markdown.css

```
@@ -0,0 +1,48 @@
+  flex: 1;
+  overflow-y: auto;
+  padding: var(--space-md);
+}
+
+.block-markdown__body .code-block {
+  background: transparent;
+}
+
+.block-markdown__body .code-block pre {
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.11.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)